### ISSUE TITLE
linebreak-style: "system" getting overridden by .editorconfig "LF"

### ISSUE THREAD

**Issue by jacobq** [ember-template-lint/ember-template-lint#2340]
I'm getting all sorts of  `error  Wrong linebreak used. Expected LF but found CRLF  linebreak-style` when on Windows despite having specified `system` explicitly. It appears that this rule gives `.editorconfig` preference, which is problematic for me. Isn't it common practice to let git checkout do EOL conversions (`autocrlf = true`)? AFAIK, `.editorconfig` does not provide an "auto" / "system" option for EOL, so on Windows I just don't worry about it. It only seems to be a problem for the linter...I suppose I could disable the linting rule, but I would've thought that `.editorconfig` should've only applied as a default, not when explicitly supplying another option. Is there a broader consensus around this?

```js
// .template-lintrc.js
module.exports = {
  extends: ['recommended', 'stylistic'],
  rules: {
    /* ... */
    "linebreak-style": "system", // want this to override .editorconfig (LF)
  },
};
```

```ini
// .editorconfig

root = true

[*]
end_of_line = lf
# ...
```

See also #490

**Comment by lifeart** [comment]
Closing issue, but it's still listed here: https://github.com/ember-template-lint/ember-template-lint/issues/3209
